create-cardholders-and-cards/server/dotnet/Program.cs

```diff
@@ -1,142 +1,145 @@
-using System.Text.Json;
-using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.StaticFiles.Infrastructure;
 using Microsoft.Extensions.FileProviders;
-
 using Stripe;
 using Stripe.Issuing;
+using CardCreateOptions = Stripe.Issuing.CardCreateOptions;
+using CardService = Stripe.Issuing.CardService;
+
+// For sample support and debugging, not required for production:
+StripeConfiguration.AppInfo = new AppInfo
+{
+    Name = "stripe-samples/issuing/create-cardholder-and-card",
+    Url = "https://github.com/stripe-samples",
+    Version = "0.0.1",
+};
 
 var builder = WebApplication.CreateBuilder(args);
+builder.Configuration.AddEnvironmentVariables();
+
+var stripeKey = builder.Configuration["STRIPE_SECRET_KEY"];

```

Comment:
Don't use env vars directly, instead go through the configuration infrastructure so all fallbacks like app.config work.

Reply:
For all the other languages, we use `.env` files to load the environment variables. I noticed you removed DotNetEnv here. Is there a way to load the `.env` file using the built in configuration infra or should I still use that other package?

Reply:
Unfortunately there is no built in way: https://github.com/dotnet/runtime/issues/49667 

We can try using the DotNetEnv and integrating it into the configuration system using the in-memory provider:

```
builder.Configuration.AddInMemoryCollection(DotNetEnv.Env.TraversePath().Load().ToDictionary());
```

Reply:
I got it to work like this:

```cs
DotNetEnv.Env.Load();

var builder = WebApplication.CreateBuilder(args);
builder.Configuration.AddEnvironmentVariables();

// Setup API keys
var stripeSecretKey = builder.Configuration["STRIPE_SECRET_KEY"];
var stripePublishableKey = builder.Configuration["STRIPE_PUBLISHABLE_KEY"];
var stripeWebHookSigningSecret = builder.Configuration["STRIPE_WEBHOOK_SECRET"];
```

How's this?